net/WebSocketHandler.hpp

```diff
@@ -221,8 +226,8 @@ class WebSocketHandler : public ProtocolHandlerInterface
 
     void shutdown(bool goingAway, const std::string &statusMessage) override
     {
-        shutdown(goingAway ? WebSocketHandler::StatusCodes::ENDPOINT_GOING_AWAY :
-                 WebSocketHandler::StatusCodes::NORMAL_CLOSE, statusMessage);
+        shutdownImpl(goingAway ? WebSocketHandler::StatusCodes::ENDPOINT_GOING_AWAY :
+                     WebSocketHandler::StatusCodes::NORMAL_CLOSE, statusMessage, false, false);
```

Comment:
Commented argument-names is always welcome when we have arbitrary arguments like `false`. It helps readability.

Reply:
OK. Yes, `bool` in particular is also dangerous (cast, change signature). Glad this is only a private implementation method. 
Adding the comments .. these days I am split about it as toolchains do that. But OK.